index.js

```diff
@@ -291,12 +294,27 @@ demo`;
         modulesGet(id);
         break;
 
+      case "archive":
```

Comment:
do we have a list of actions somewhere?

Reply:
We have a top level help command (that was missing `modules`), as well as help for individual actions `cbx modules help`